src/test/java/com/google/sps/GameTest.java

```diff
@@ -0,0 +1,76 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import com.google.sps.utils.Game;
+import com.google.sps.utils.UserUtils;
+import java.util.ArrayList;
+
+// tests the usergame class functions 
+@RunWith(JUnit4.class)
+public final class GameTest {
+    private final LocalServiceTestHelper helper = new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+
+    @Before
+    public void setUp() {
+    helper.setUp();
+    }
+
+    @After
+    public void tearDown() {
+    helper.tearDown();
+    }
+
+    @Test 
+    //Tests if the properties are set right for the game class 
+    public void SetsGameEntityCorrectly() {
+        Game game = new Game();
+        game.setGameId("1234");
+        game.setGameName("Plants");
+        game.setQuizQuestion("Which plant looks the healthiest?");
+        game.setQuizTimestamp(1594309443653L);
+        ArrayList<String> users = new ArrayList<String>();
```

Comment:
Please use a non-empty list, so it's clear that `game.getUserIds()` isn't returning a default value.

Reply:
Underneath line 53 I add users to this list so that it is not empty is that what you mean?